frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -435,17 +434,51 @@ export function NeuronInferralModal({ handleClose }: Props) {
   );
 }
 
-export function ApplyMergerModeModal({ handleClose }: Props) {
+type MaterializeVolumeAnnotationModalProps = Props & {
+  selectedVolumeLayer?: APIDataLayer;
+};
+
+export function MaterializeVolumeAnnotationModal({
+  selectedVolumeLayer,
+  handleClose,
+}: MaterializeVolumeAnnotationModalProps) {
   const dataset = useSelector((state: OxalisState) => state.dataset);
   const tracing = useSelector((state: OxalisState) => state.tracing);
   const activeSegmentationTracingLayer = useSelector(getActiveSegmentationTracingLayer);
+  const fixedSelectedLayer = selectedVolumeLayer || activeSegmentationTracingLayer;
+  const isMergerModeEnabled = useSelector(
+    (state: OxalisState) => state.temporaryConfiguration.isMergerModeEnabled,
+  );
+  let description = (
+    <p>
+      Start a job that takes the current state of this volume annotation and materializes it into a
+      new dataset. All annotations done on the
+      {` "${fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing)}" `}
+      volume layer will be merged with the data of the fallback layer.
```

Comment:
Again, I would expect that not every annotation has a fallback layer or am I wrong about that?

Reply:
I changed the code so that 
> All annotations done on the
      {` "${fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing)}" `}
      volume layer will be merged with the data of the fallback layer.

will only be shown when there is a fallback layer.